test/e2e/framework/framework.go

```diff
@@ -333,7 +333,7 @@ func (f *Framework) AfterEach() {
 		if len(nsDeletionErrors) != 0 {
 			messages := []string{}
 			for namespaceKey, namespaceErr := range nsDeletionErrors {
-				messages = append(messages, fmt.Sprintf("Couldn't delete ns: %q: %s", namespaceKey, namespaceErr))
+				messages = append(messages, fmt.Sprintf("Couldn't delete ns: %q: %s (%#v)", namespaceKey, namespaceErr, namespaceErr))
```

Comment:
We are for the string, see [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/util/errors/errors.go#L64-L66), it looks like here the actual error is being swallowed somewhere.